### PR TITLE
prettify distributions page

### LINKED ISSUE #308: prettify distributions page

### PR DESCRIPTION
**Pull request by blukai**
fixes #308 
Modified `abbreviateNumber`, `Heading`, `Warning`

### CODE REVIEW

**Comment by blukai** [comment]
> The CDN caches paths permanently, so instead of master you should use a release version or commit hash.

I know, but I didn't think `react-flags` will be restructured. It keeps its structure from initial commit. https://github.com/wiredmax/react-flags/tree/4f657e1071d4e30bfdcc7ebd30f01dbe4a073f5f

**Comment by howardchung** [comment]
What if a country changes their flag?

**Comment by blukai** [comment]
Idk, actually I found a mismatch on Saint Martin flag in react-flags dependency. Probably I might use svg flags from here https://github.com/lipis/flag-icon-css/tree/master/flags/4x3 that uses current ui. Seems like flag-icon-css has updates and a lot of stars 🍀

File: src/lang/en.json [hunk 1]
```diff
@@ -36,9 +37,11 @@
   "benchmarks_higher_than": "is equal or higher than",
   "benchmarks_recent_performances": "of recent performances on this hero",
 
-  "distributions_mmr": "Solo MMR",
-  "distributions_country_mmr": "Solo MMR By Country",
-  "distributions_warning_1": "Warning! This data set is limited to players displaying MMR on profile and sharing public match data.",
+  "distributions_heading_mmr": "Solo MMR distributions",
+  "distributions_heading_country_mmr": "Average MMR by country",
+  "distributions_tab_mmr": "Solo MMR",
```
Comment (howardchung) [inline src/lang/en.json]: could we just reuse the headings for tabs?

File: src/components/Distributions/Distributions.jsx [hunk 2]
```diff
@@ -4,32 +4,76 @@ import {
...
+      case 'zr':
+        image = '//upload.wikimedia.org/wikipedia/commons/5/5c/Flag_of_Zaire.svg';
+        name = 'Zaire';
+        break;
+      default:
+        image = `${require(`flag-icon-css/flags/4x3/${code}.svg`)}`; // eslint-disable-line global-require
+        name = row.common;
+    }
+
+    if (code === 'bq') name = 'Caribbean Netherlands';
```
Comment (howardchung) [inline src/components/Distributions/Distributions.jsx]: use braces even on single lines

**Pull request merged**